<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KitFicha</class>
 <widget class="QDialog" name="KitFicha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ficha de kit</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="edtCodKit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Código</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Descripción</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="edtNombre">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maxLength">
        <number>150</number>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QToolButton" name="btnAniadirProd">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../iconos.qrc">
         <normaloff>:/iconos/iconos/Add.ico</normaloff>:/iconos/iconos/Add.ico</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QToolButton" name="btnEliminarProd">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../iconos.qrc">
         <normaloff>:/iconos/iconos/Delete.ico</normaloff>:/iconos/iconos/Delete.ico</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="4" colspan="3">
      <widget class="QTableWidget" name="tableDetalles">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImprimir">
       <property name="text">
        <string>Imprimir</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtNombre</tabstop>
  <tabstop>btnAniadirProd</tabstop>
  <tabstop>btnEliminarProd</tabstop>
  <tabstop>edtCodKit</tabstop>
  <tabstop>tableDetalles</tabstop>
 </tabstops>
 <resources>
  <include location="../iconos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KitFicha</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KitFicha</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
